### ISSUE TITLE
Inbrowser False flag doesn't work

### ISSUE THREAD

**Issue by 6Morpheus6** [jhj0517/Whisper-WebUI#275]
**Which OS are you using?**
Windows 10 / Pinokio / Opera

Hi,
I wrote a new installer for Pinokio since the old version is broken.
Whisper starts in Pinokio and the browser at the same time and it's not possible to make it stop automatically starting in the browser.

I tested both ways:
The `--inbrowser False` flag is not working while default is set to True (on line 389)
```
parser.add_argument('--inbrowser', type=bool, default=True, nargs='?', const=True, help='Whether to automatically start
Gradio app or not')
```
But `--inbrowser` True is working while default is set to False

I tested it with Pinokio and in Anaconda Powershell. The behavior is the same.
![image](https://github.com/user-attachments/assets/3ed83cb4-731c-496f-818b-ccaed66696dd)

**Comment by jhj0517** [comment]
Hi @6Morpheus6, Thanks for reporting!

This is fixed in #277. Please feel free to reopen if you encounter any issues!

**Comment by 6Morpheus6** [comment]
Thanks a lot for the quick response @jhj0517 
This fixed the issue. :)
